src/System.IO.FileSystem/src/win/project.json

```diff
@@ -1,28 +1,33 @@
 {
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1",
+    "System.Collections": "4.0.10",
+    "System.Diagnostics.Contracts": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.10",
+    "System.Diagnostics.Tools": "4.0.0",
+    "System.IO": "4.2.0-beta-devapi-24407-01",
+    "System.IO.FileSystem.Primitives": "4.0.0",
+    "System.Resources.ResourceManager": "4.0.0",
+    "System.Runtime": "4.0.20",
+    "System.Runtime.Extensions": "4.0.10",
+    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime.InteropServices": "4.0.20",
+    "System.Text.Encoding": "4.0.10",
+    "System.Text.Encoding.Extensions": "4.0.10",
+    "System.Threading": "4.0.10",
+    "System.Threading.Overlapped": "4.0.0",
+    "System.Threading.Tasks": "4.0.10"
+  },
   "frameworks": {
-    "netstandard1.3": {
-      "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.0.1",
-        "System.Collections": "4.0.10",
-        "System.Diagnostics.Contracts": "4.0.0",
-        "System.Diagnostics.Debug": "4.0.10",
-        "System.Diagnostics.Tools": "4.0.0",
-        "System.IO": "4.0.10",
-        "System.IO.FileSystem.Primitives": "4.0.0",
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.20",
-        "System.Runtime.Extensions": "4.0.10",
-        "System.Runtime.Handles": "4.0.0",
-        "System.Runtime.InteropServices": "4.0.20",
-        "System.Text.Encoding": "4.0.10",
-        "System.Text.Encoding.Extensions": "4.0.10",
-        "System.Threading": "4.0.10",
-        "System.Threading.Overlapped": "4.0.0",
-        "System.Threading.Tasks": "4.0.10"
-      },
+    "netstandard1.7": {
       "imports": [
         "dotnet5.4"
```

Comment:
>"dotnet5.4" [](start = 8, length = 11)

I believe that this should be dotnet5.8 and the one bellow should be dotnet5.4. @ericstj is that right?